jetty-server/src/main/java/org/eclipse/jetty/server/session/AbstractSessionCache.java

```diff
@@ -125,7 +126,7 @@
      * @param session the session object
      * @return null if the session wasn't already in the map, or the existing entry otherwise
      */
-    public abstract Session doPutIfAbsent(String id, Session session);
+    protected abstract Session doPutIfAbsent(String id, Session session);
```

Comment:
changing visibility in minor release? (maybe in 10?)

Reply:
See previous comment. I guess if I absolutely had to, I could leave the viz changes until 10 ...